README.md

```diff
@@ -42,7 +43,7 @@ using ConsentManagementProviderLib;
 ```c#
 private void Start()
 {
-    CMP.LoadMessage(authId: "JohnDoe");
+    CMP.LoadMessage(authId: "JohnDoe"); // it may be null
```

Comment:
```diff
-      CMP.LoadMessage(authId: "JohnDoe"); // it may be null
+     CMP.LoadMessage(authId: null); // or pass it a String if you wish to use authenticated consent
```